### ISSUE TITLE
API call to get a more accurrate description of the Field of View

### ISSUE THREAD

**Issue by imbasimba** [cds-astro/aladin-lite#101]
As it is now (or at least in Aladin Lite 2), I could only get the Field of View (FoV) from extracted from Aladin in one number. To be more accurate, I would like an API call to get both the horizontal **and** vertical FoV.  Also, I would like Aladin Lite to fire off events when they change.

**Comment by ManonMarchand** [comment]
I would love that too, could be relevant with the new height parameter in ipyaladin

**Comment by ManonMarchand** [comment]
getFov is actually already in the API (see https://aladin.cds.unistra.fr/AladinLite/doc/API/)

Then the events is an other story

**Comment by bmatthieu3** [comment]
Indeed, it seems to be already implemented in the v3, the getFoV methods return an array containing 2 elements (the width and height fov). Maybe we can provide a getFovWidth and  a getFovHeight method to get them separately too. For the events I see that when you resize the view along a direction it changes the vertical fov and keeps the horizontal one the same. Appart from that, I do not know other such cases.

**Comment by ManonMarchand** [comment]
Renamed the issue to fit the second item

**Comment by bmatthieu3** [comment]
Hi @pmatsson, @imbasimba - may I ask what is your use case for listening to a fov event ?

**Comment by pmatsson** [comment]
Hi, @bmatthieu3  I think this issue can be closed with no further action from your side. Listening to the existing zoomChanged event and extracting the FoV with the existing getFoV() method is enough for our use case.

**Comment by bmatthieu3** [comment]
thanks @pmatsson, I close this
